README.md

```diff
@@ -1 +1,23 @@
-# hw01
\ No newline at end of file
+# Adam Fofana
+
+![Alt text is fun](IMG_2520.jpg)
+
+
+
+## Education
+BA in Government and Politics with a Minor in Linguistics from the University of Maryland (May 2019)
+
+MA in International Relations from the University of Chicago (Expected June 2020)
+
+## Academic Interests
+1. Economic Integration in West Africa
+2. International Organizations
+3. Public Transportation
+4. Urbanism
+
+## Personal Interests
+1. Running (**>800 meters**)
+2. Comicbooks
+3. Marvel Movies
+4. Science Fiction
```

Comment:
Very cool! (There's a comicbook shop in Hyde Park called First Aid Comics- the more you know!)